### DISCUSSION TITLE
Update-2023-08-10 Sprites

### DISCUSSION THREAD

**Discussion by kaveh808** [kaveh808/kons-9#240] [Announcements]
Implemented a basic `SPRITE-INSTANCER` class. Sprites are shapes which always face the camera.

Sprites from particles:
![Screen Shot 2023-08-10 at 9 38 16 AM](https://github.com/kaveh808/kons-9/assets/103860793/57a4ca76-c41b-4778-91f6-e88f7b58257b)

Sprites at polyhedron points:
![Screen Shot 2023-08-10 at 9 40 08 AM](https://github.com/kaveh808/kons-9/assets/103860793/178244ed-1b9c-41da-9198-96dd09bcfa83)

Sprite transparency interacting with other scene shapes:
![Screen Shot 2023-08-10 at 9 39 33 AM](https://github.com/kaveh808/kons-9/assets/103860793/5874e798-f07f-4cfc-86a5-021a8d50d157)

**Comment by aykaramba** [comment]
Lovely!

**Comment by kayomarz** [comment]
I agree with @aykaramba. These are really creative and cool!
@kaveh808 apart from adding features to kons-9 it would be great if a couple of contributors would like to collaborate and build a tiny interactive game or application. any ideas? it would be fun.

**Reply by kaveh808** [reply]
That sounds like a promising idea. Why don't you start a discussion thread?

**Reply by kayomarz** [reply]
Great, I will start a discussion.
